jda/src/main/java/co/aikar/commands/JDACommandManager.java

```diff
@@ -247,11 +249,19 @@ void dispatchEvent(MessageReceivedEvent event) {
                 break;
             }
         }
+        if (prefixFound == null && config.mentionPrefixEnabled()) {
+            if (msg.startsWith("<@" + selfUserId + ">")) {
+                prefixFound = "<@" + selfUserId + ">";
+            } else if (msg.startsWith("<@!" + selfUserId + ">")) {
+                prefixFound = "<@!" + selfUserId + ">";
+            }
+        }
         if (prefixFound == null) {
             return;
         }
 
-        String[] args = ACFPatterns.SPACE.split(msg.substring(prefixFound.length()), -1);
+        // str.replaceAll("^[ \t]+", "") - works like String.stripLeading() from Java 11
+        String[] args = ACFPatterns.SPACE.split(msg.substring(prefixFound.length()).replaceAll("^[ \\t]+", ""), -1);
```

Comment:
I would say that this pattern should be cached

Reply:
Ok, now I'm using ```ACFPatterns.getPattern()``` which caches patterns